mods/ts/rules/shared-support.yaml

```diff
@@ -9,6 +9,7 @@ NAPULS:
 		Queue: Defense
 		BuildPaletteOrder: 90
 		Prerequisites: radar
+		BuildLimit: 1
```

Comment:
Partly because it doesn't work with more than one emp cannon, but also because that's what was requested in #7349